docs/platforms.rst

```diff
@@ -0,0 +1,51 @@
+.. _platforms:
+
+Platforms
+=========
+
+Bottery design always considered the possibility to integrate to any chat platform. At the code level, there're many abstractions to make it possible. Bottery works out of the box with two chat platforms, Facebook Messenger and Telegram but it give the tools to easly integrate to any specific platform.
+
+All of this platforms has different needs and different settings, but the views, when written without using specific platform features, should work for all of them.
+
+Operation Modes
+---------------
+
+The first part of Bottery operation is when it get new messages. And Bottery do it throut two different ways: polling and webhook;
```

Comment:
The first part of Bottery operation is when `(and how)` it **gets** new messages. 

The second phrase: "Bottery can receive messages in two different ways"